tests/bson/bson-utcdatetime-int-size.phpt

```diff
@@ -1,5 +1,7 @@
 --TEST--
 Test for UTCDateTime and integer parsing
+--SKIPIF--
+<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM implements only an older version of DateTime"); ?>
```

Comment:
Shouldn't this rather check for correct DateTime behavior, so the test can start working when they fix DateTime?
Otherwise this will never be enabled, even when their DateTime is updated, possibly leading to bugs.